dts/arm/nuvoton/npcx7m6fb.dtsi

```diff
@@ -4,693 +4,20 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <arm/armv7-m.dtsi>
-/* Macros for device tree declarations */
-#include <dt-bindings/clock/npcx_clock.h>
-#include <dt-bindings/espi/npcx_espi.h>
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/i2c/i2c.h>
-#include <dt-bindings/pwm/pwm.h>
-
-/* NPCX7 series pinmux mapping table */
-#include "npcx/npcx7-alts-map.dtsi"
-/* NPCX7 series mapping table between MIWU wui bits and source device */
-#include "npcx/npcx7-miwus-wui-map.dtsi"
-/* NPCX7 series mapping table between MIWU groups and interrupts */
-#include "npcx/npcx7-miwus-int-map.dtsi"
-/* NPCX7 series eSPI VW mapping table */
-#include "npcx/npcx7-espi-vws-map.dtsi"
-/* NPCX7 series low-voltage io controls mapping table */
-#include "npcx/npcx7-lvol-ctrl-map.dtsi"
+#include <mem.h>
+#include "npcx7.dtsi"
 
 / {
-	cpus {
-		#address-cells = <1>;
-		#size-cells = <0>;
-
-		cpu@0 {
-			device_type = "cpu";
-			compatible = "arm,cortex-m4f";
-			reg = <0>;
-		};
+	flash0: flash@10090000 {
+		reg = <0x10090000 DT_SIZE_K(192)>;
 	};
 
-	flash0: flash@10090000 {
-		reg = <0x10090000 0x30000>;
+	flash1: flash@64000000 {
+		reg = <0x64000000 DT_SIZE_M(1)>;
```

Comment:
Is this correct? Looking at the flash regions defined in the Chrome EC code base, the chip variant CHIP_VARIANT_NPCX7M6FB  is configured for only 512K flash.

Reply:
I have checked with the datasheet & our PM.
For the NPCX7 chip, 
NPCX79nxB flash size is 1MB
NPCX79nxC flash size is 512KB.
I think the setting is correct.
For the Chrome EC code, NPCX7M6FB code ram size is 192KB. RO+RW don't exceed 512KB. It can directly use the original flash layout setting. NPCX7M7FB code ram size is 320KB so it can't use the same setting.

Reply:
Sounds good.